python/tvm/relax/frontend/torch/exported_program_translator.py

```diff
@@ -1149,14 +1170,22 @@ def from_exported_program(
         from torch import fx  # type: ignore
 
         # Create input variables.
-        parameter_buffer_constant_vars, user_input_vars = self.create_input_vars(exported_program)
+        (
+            parameter_buffer_constant_vars,
+            user_input_vars,
+            range_constraints,
+        ) = self.create_input_vars(exported_program)
         inputs_vars = user_input_vars.copy()
         inputs_vars.update(parameter_buffer_constant_vars)
 
         # Initialize the block builder with a function and a dataflow block.
         self.block_builder = relax.BlockBuilder()
         func_name = "main"
         func_attrs = {"num_input": len(user_input_vars)} if keep_params_as_input else None
+        if range_constraints:
+            if func_attrs is None:
+                func_attrs = {}
+            func_attrs["shape_var_constraints"] = range_constraints
```

Comment:
Please use `tir_var_upper_bound` to annotate upper bound.
I don't think we need to keep the lower bound at the moment. If we have a real use case for it, it's fine to keep it in the Relax module.
https://github.com/apache/tvm/blob/main/src/relax/transform/static_plan_block_memory.cc#L62-L66